.github/workflows/pr-close-image-delete.yaml

```diff
@@ -0,0 +1,40 @@
+name: Delete quay image on PR closed
+on:
+  pull_request_target:
+    types: [closed]
+    branches:
+      - "main"
+    paths-ignore:
+      - "LICENSE"
+      - "**/.gitignore"
+      - "**.md"
+      - "**.adoc"
+      - "*.txt"
+      - ".github/**"
+      - "docs/**"
+      - "demo/**"
+
+jobs:
+  delete-pr-quay-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: "0"
+      - name: Install skopeo
+        shell: bash
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install skopeo
+      - name: Login to quay.io
+        env:
+          QUAY_TOKEN: ${{ secrets.QUAY_ROBOT_TOKEN }}
+          QUAY_ROBOT_USERNAME: ${{ secrets.QUAY_ROBOT_USERNAME }}
```

Comment:
Note: these secrets have to be added to this repo.

Reply:
@anishasthana I don't seem to have permissions to view the tokens and/or create a new robot account on quay in order to add the quay token to this repo. Could you create a new robot account for this repo's github action (write access) and add it to the actions secret for the repo? Thanks.